src/main/java/teammates/ui/automated/InstructorCourseJoinEmailWorkerAction.java

```diff
@@ -39,7 +39,7 @@ public void execute() {
         CourseAttributes course = logic.getCourse(courseId);
         Assumption.assertNotNull(course);
 
-        InstructorAttributes instructor = logic.getInstructorForEmail(courseId, instructorEmail);
+        InstructorAttributes instructor = logic.getInstructorById(courseId, instructorEmail);
```

Comment:
What if instructor A invites instructor B and A finds that B's email is wrong while the instructor account is already created. A changes the email accordingly and resend the invitation email. In this case, I believe B will never receive the email.

Actually, if it is not to complicated, we can implement the re-creation logic same as the student DB.

Reply:
> A changes the email accordingly and resends the invitation email. 

This is not possible. Once B is added by A, A can only edit B's email address once B has joined using a Google account.

<img width="1149" alt="screen shot 2018-07-01 at 1 46 54 pm" src="https://user-images.githubusercontent.com/12574756/42131472-4b5aa608-7d35-11e8-8734-29a96106c6b0.png">


> Actually, if it is not to complicated, we can implement the re-creation logic same as the student DB.

I will still investigate this as in future its better to adopt this strategy cause currently our database is flawed as there will be many cases where `emaild` in the `id` is not same as `emailId` feild.

Reply:
Okay, please add some comment in InstructorCourseJoinEmailWorkerAction.java to explain the situation.

Reply:
Added. Ready for review @xpdavid